### PR TITLE
OKTA-676888 - Add granular authenticator control

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Updates as per granular authenticator control
- **Is this PR related to a Monolith release?** 2024.01.2

### Resolves:

* [OKTA-676888 ](https://oktainc.atlassian.net/browse/OKTA-676888 )

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/03cf475dac306cd5187aaf91f5a8141776449246/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 69 | 512 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/356a988a-1f9c-4261-83df-4084cd0d8bb9) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.2935bb49-5605-43ae-9f1c-222b41038fdc?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 1]
```diff
@@ -2374,8 +2374,20 @@ The number of Authenticator class constraints in each Constraint object must be
...
+| `authenticationMethods`  <ApiLifecycle access="ea" />  | array of [Authentication method objects](#authentication-method-object) | This property specifies the precise authenticator and method for authentication.  || `OPTIONAL`|
+| `excludedAuthenticationMethods` <ApiLifecycle access="ea" />   | array of [Authentication method objects](#authentication-method-object) | This property specifies the precise authenticator and method to exclude from authentication.  || `OPTIONAL`|
+| `required` <ApiLifecycle access="ea" />   | Boolean | This property indicates whether the knowledge or possession factor is required by the assurance. It's optional in the request, but is always returned in the response. By default, this field is `true`. If the knowledge or possession constraint has values for`excludedAuthenticationMethods` the `required` value is false.  || `OPTIONAL`|
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:2381]:
```suggestion
| `required` <ApiLifecycle access="ea" />   | Boolean | This property indicates whether the knowledge or possession factor is required by the assurance. It's optional in the request, but is always returned in the response. By default, this field is `true`. If the knowledge or possession constraint has values for`excludedAuthenticationMethods`, then the `required` value is false.  || `OPTIONAL`|
```

**Review by barbaravo-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/233e8a235ead4e70fdfe77e73db29d92ea7a1d90/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 69 | 512 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7449bfdb-99d6-4b3b-a699-408b00f7c0e2) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.a80cafec-5d36-481f-a3fd-16cae3413979?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/reference/api/policy/index.md](https://github.com/okta/okta-developer-docs/blob/923819ec6ea2df5168933182ccc0911f19df7294/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Fapi%2Fpolicy%2Findex.md) | 69 | 512 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/73243526-7cd5-4d88-a98f-d81bcf55ac47) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 1 of 1 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.108ecb7a-56bd-49d3-9dcd-9549c9316160?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Review by barbaravo-okta — APPROVED** [review]

**Comment by brianduffield-okta** [comment]
SME review approved in Slack/JIRA

**Pull request merged**